pkg/api/validation/schema.go

```diff
@@ -87,7 +88,7 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 	if kind == nil {
 		return fmt.Errorf("kind not set")
 	}
-	allErrs := s.ValidateObject(obj, apiVersion.(string), "", apiVersion.(string)+"."+kind.(string))
+	allErrs := s.ValidateObject(obj, apiutil.GetVersion(apiVersion.(string)), "", apiutil.GetVersion(apiVersion.(string))+"."+kind.(string))
```

Comment:
Suggest naming this variable groupVersion, and for clarity making a new variable `version := apiutil.GetVersion(groupVersion.(string))`.

But it's not clear to me how ValidateObject is going to work without knowing the group? And it's weird that it takes the version twice.